docs/X86_Silicon/index.mdx

```diff
@@ -0,0 +1,165 @@
+---
+sidebar_position: 2
+title: x86_64 on Apple Silicon
+tags:
+  - VM
+  - MacOS
+  - GDB
+---
+
+#### How do I get a WSL on Mac Silicon?
+
+This documentation aims to teach you how to develop on x86_64 architecture with your Silicon Mac.
+
```

Comment:
Add a paragraph to explain the purpose of the tutorial and why it could be interesting (its applications for example)